docs/src/userguide/numpy_tutorial.rst

```diff
@@ -6,35 +6,29 @@
 Cython for NumPy users
 **************************
 
-.. NOTE:: Cython 0.16 introduced typed memoryviews as a successor to the NumPy
-          integration described here.  They are easier to use than the buffer syntax
-          below, have less overhead, and can be passed around without requiring the GIL.
-          They should be preferred to the syntax presented in this page.
-          See :ref:`Typed Memoryviews <memoryviews>`.
-
-
 This tutorial is aimed at NumPy users who have no experience with Cython at
 all. If you have some knowledge of Cython you may want to skip to the
-''Efficient indexing'' section which explains the new improvements made in
-summer 2008.
+''Efficient indexing'' section.
 
 The main scenario considered is NumPy end-use rather than NumPy/SciPy
 development. The reason is that Cython is not (yet) able to support functions
-that are generic with respect to datatype and the number of dimensions in a
+that are generic with respect to the number of dimensions in a
 high-level fashion. This restriction is much more severe for SciPy development
 than more specific, "end-user" functions. See the last section for more
 information on this.
 
 The style of this tutorial will not fit everybody, so you can also consider:
 
-* Robert Bradshaw's `slides on cython for SciPy2008
-  <http://wiki.sagemath.org/scipy08?action=AttachFile&do=get&target=scipy-cython.tgz>`_
-  (a higher-level and quicker introduction)
-* Basic Cython documentation (see `Cython front page <http://cython.org>`_).
-* ``[:enhancements/buffer:Spec for the efficient indexing]``
+* Kurt Smith's `video tutorial of Cython at SciPy 2015
+  <https://www.youtube.com/watch?v=gMvkiQ-gOW8&t=4730s&ab_channel=Enthought>`_.
+  It's longuer but some readers like watching talks more than reading.
```

Comment:
"longuer" -> "longer". But actually, I'd remove this sentence all together. I don't consider it helpful in any way.